src/transformers/models/mistral/modeling_mistral.py

```diff
@@ -51,6 +53,7 @@
 
     _flash_supports_window_size = "window_size" in list(inspect.signature(flash_attn_func).parameters)
 
+_is_torch_version_greater_or_equal_than_2_2_0 = version.parse(get_torch_version()) >= version.parse("2.2.0")
```

Comment:
Let's do something close to 
https://github.com/huggingface/transformers/blob/main/src/transformers/utils/import_utils.py#L661

to make sure it's re-usable!

Reply:
> Let's do something close to https://github.com/huggingface/transformers/blob/main/src/transformers/utils/import_utils.py#L661
> 
> to make sure it's re-usable!

you're right, and i've added this function to import_utils for convenience， please review it again @ArthurZucker